### ISSUE TITLE
Enhance components of common with screen reader attributes

### ISSUE THREAD

**Issue by Larisa-Staroverova** [SAP/spartacus#14158]
Verify whether the screen reader reads everything as expected
Clarify what and how should be read with PO / UX
Enhance /add `aria` accordingly

List of components:
- [x] configurator-cart-entry-bundle-info
- [x] configurator-cart-entry-info
- [x] configurator-issues-notification
- [x] configure-cart-entry
- [x] configure-product
